File: passenger-service/src/main/java/com/software/modsen/passengerservice/config/ModelMapperConfig.java
package com.software.modsen.passengerservice.config;

import lombok.RequiredArgsConstructor;
import org.modelmapper.ModelMapper;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
@RequiredArgsConstructor
public class ModelMapperConfig {
    @Bean
    public ModelMapper modelMapperBean() {
        return new ModelMapper();
    }
}

File: passenger-service/src/main/kotlin/com/software/modsen/passengerservice/config/ModelMapperConfig.kt
package com.software.modsen.passengerservice.config

import org.modelmapper.ModelMapper
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class ModelMapperConfig {
    @Bean
    open fun modelMapperBean(): ModelMapper {
        return ModelMapper()
    }
}